helm/chaos-mesh/values.yaml

```diff
@@ -48,7 +48,8 @@ controllerManager:
 chaosDaemon:
   image: pingcap/chaos-daemon:latest
   imagePullPolicy: Always
-  port: 31767
+  grpcPort: 31767
+  httpPort: 31766
```

Comment:
Maybe we should remove the httpPort config. I think we don't need to expose this config and it is ok to hard code it in the yaml.